galley/pkg/server/settings/args.go

```diff
@@ -108,6 +108,9 @@ type Args struct { // nolint:maligned
 	// Enable service discovery / endpoint processing.
 	EnableServiceDiscovery bool
 
+	// Enable Config Analysis service, that will analyze and update CRD status. UseOldProcessor must be set to false.
```

Comment:
Should we log an error or warning if EnableConfigAnalysis and UseOldProcessor are both true?

Reply:
At some point the UseOldProcessor should simply go away (1.5 probably).